test/screenshot/lib/controller.js

```diff
@@ -16,6 +16,7 @@
 
 'use strict';
 
+const child_process = require('child_process'); // eslint-disable-line
```

Comment:
Why do you need `// eslint-disable-line` here?

Reply:
Oops! Forgot to include the specific rule I'm disabling: `camelcase`. Added!

I suppose I could rename the variable to `childProcess`, but that felt kinda weird. Every example I've seen calls it `child_process`. 🤷‍♂️ 

WDYT?

Reply:
Unless you need the whole `child_process` module, you could could destructure and just load `spawnSync`.

That being said, I'd just camel case it.